### PR TITLE
Explore: Scroll split panes in Explore independently

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
First step of logs navigation feature - ensuring that split panes are scrolling independently. 
Before:
https://user-images.githubusercontent.com/30407135/114691747-478b0080-9d18-11eb-99ba-6ce12d773d02.mov

After:
https://user-images.githubusercontent.com/30407135/114691726-435ee300-9d18-11eb-9db9-aea748bd055f.mov

### CODE REVIEW

File: public/app/features/explore/Wrapper.tsx [hunk 1]
```diff
@@ -31,18 +31,16 @@ export class Wrapper extends Component<WrapperProps> {
 
     return (
       <div className="page-scrollbar-wrapper">
-        <CustomScrollbar autoHeightMin={'100%'}>
-          <div className="explore-wrapper">
+        <div className="explore-wrapper">
```
Comment (ivanahuckova) [inline public/app/features/explore/Wrapper.tsx:34]: This change highlighting is weird. I just removed `<CustomScrollbar/>`.

Comment (torkelo) [reply public/app/features/explore/Wrapper.tsx:34]: Where is CustomScrollbar now then? Without it looks pretty bad in firefox on linux & ubuntu

Comment (ivanahuckova) [reply public/app/features/explore/Wrapper.tsx:34]: It is in `public/app/features/explore/Explore.tsx` (in each individual pane).

Comment (gabor) [reply public/app/features/explore/Wrapper.tsx:34]: @ivanahuckova at the top of the github-diff you can switch "ignore-whitespace: ON", then the diff looks better. i think the reason is that the indent changed for a lot of lines, causing the large visual diff.

**Review by gabor — APPROVED** [review]
looks good to me :+1:

**Comment by ifrost** [comment]
n

**Review by ifrost — APPROVED** [review]

**Comment by bhanu8824** [comment]
I would like to have the ability to scroll each pane in the split view independently. This would allow for more effective comparisons and analysis of different datasets within Grafana Explore.

Enabling independent scrolling for split panes would greatly enhance the usability of the Explore feature for data analysis. If there is already a way to achieve this, could someone please provide guidance on how to enable it?
@ivanahuckova can you help me on this?

**Pull request merged**